### ISSUE TITLE
Change default paper size for export (svg/png/pdf)

### ISSUE THREAD

**Issue by averissimo** [gephi/gephi#1453]
**It would be a very cool new feature to be able to set default paper size on exports. (image below on the existing dialog that requires a "set this as default" option**

Use case:
- Exporting multiple work-spaces one by one and iterating on it becomes a slow and painful experience, making sure a common paper size is used on all exports that differs from A4 standard

This happened to me while exporting multiple graphs for a paper that needed to be of the same size (actually the requirement is the ratio). I ended up changing the code (very crude) and compiling my custom Gephi:

https://github.com/averissimo/gephi/commit/7a083f14e579fa8edec7750bc9ee1f49319f048c

![selection_335](https://cloud.githubusercontent.com/assets/211358/14359762/dfd82918-fcea-11e5-8021-2c8ec88e772b.png)

**Comment by Splines** [comment]
Maybe helpful for viewers of this post: If you use the Gephi Toolkit, you might want to use the PDFExporter, see [here](https://github.com/gephi/gephi-toolkit-demos/blob/master/src/main/java/org/gephi/toolkit/demos/ImportExport.java#L108-L114):

```java
//PDF Exporter config and export to Byte array
PDFExporter pdfExporter = (PDFExporter) ec.getExporter("pdf");
pdfExporter.setPageSize(PageSize.A0);
pdfExporter.setWorkspace(workspace);
ByteArrayOutputStream baos = new ByteArrayOutputStream();
ec.exportStream(baos, pdfExporter);
byte[] pdf = baos.toByteArray();
```

**Comment by mbastian** [comment]
Thanks for the report. I'm closing as we've resolved other issues like #2671 that wouldn't make this necessary I think. Now the export settings are remembered between use so if you apply it once and then keep exporting it should work. Those fix will be included in the upcoming 0.10.0 release.
